packages/graph/hash_graph/lib/graph/src/store/postgres/ontology/read.rs

```diff
@@ -24,23 +24,17 @@ use crate::{
 #[async_trait]
 impl<C: AsClient, T> Read<T> for PostgresStore<C>
 where
-    T: for<'q> PersistedOntologyType<
-            Inner: PostgresQueryRecord<Path<'q>: Debug + Send + Sync + OntologyPath>
-                       + OntologyDatabaseType
-                       + TryFrom<serde_json::Value, Error: Context>
-                       + Send
-                       + 'static,
-        > + Send,
+    for<'q> T: PersistedOntologyType<
```

Comment:
Is it to be able to use it in `PersistedOntologyType` and `PostgresQueryRecord`?

Reply:
The `for<'q>` is required for `PostgresQueryRecord<Path<'q>: ...>`. I chose to put the `for<'q>` before `T` to make it more readable. This enables us to reuse the lifetime if needed. Happy to change back if desired.

Reply:
I'm not sure it is more readable, as if I was trying to look for what `T` is, it's harder to immediately find because its indentation is messed up and has stuff in front of it.

That said, I appreciate that this syntax would be useful/required in other situations so perhaps we should try and standardise on always doing this rather than mixing.

Reply:
Honestly, I don't think this is more readable:
```rust
where
    T: PersistedOntologyType<
            Inner: OntologyDatabaseType + TryFrom<serde_json::Value, Error: Context>,
        > + for<'q> PostgresQueryRecord<Path<'q>: Debug + Send + Sync + OntologyPath>
        + Send
        + 'static,
```

Reply:
I moved out the `Inner`:
```rust
where
    T: PersistedOntologyType
        + for<'q> PostgresQueryRecord<Path<'q>: Debug + Send + Sync + OntologyPath>
        + Send
        + 'static,
    T::Inner: OntologyDatabaseType + TryFrom<serde_json::Value, Error: Context>,
```

Reply:
Note: we discussed offline to rename `Inner`